### DISCUSSION TITLE
ConvexDecomposition asserts and invalid results

### DISCUSSION THREAD

**Discussion by helpsl** [bulletphysics/bullet3#3549] [General]
When generating a decomposition in 2.86.1 it produces a few normalisation asserts and does not correctly decompose the object (Works in 2.85.1).

Looking at it line concavity.cpp:708 has been ifdef'd out to avoid a warning about an if(0.. statement.
However, there is an else statement which has also been included in this, which I assume wasn't meant to be.

**Comment by AndresTraks** [comment]
A great example of why consistent formatting is needed. :) #1007 
It's pretty hard to follow an unformatted method that is also 225 lines long.
